boefjes/tests/conftest.py

```diff
@@ -61,22 +62,22 @@ def pop_item(self, queue: str) -> Optional[QueuePrioritizedItem]:
         except IndexError:
             raise self.raise_on_empty_queue
 
-    def patch_task(self, task_id: str, status: TaskStatus) -> None:
+    def patch_task(self, task_id: UUID, status: TaskStatus) -> None:
         with self.log_path.open("a") as f:
-            f.write(f"{task_id},{status.value}\n")
+            f.write(f"{str(task_id)},{status.value}\n")
```

Comment:
Aren't objects other than strings implicitly cast to a `str` within f-strings?

Reply:
Yes this is probably redundant

Reply:
Fixed